extensions/amp-install-serviceworker/0.1/test/test-amp-install-serviceworker.js

```diff
@@ -255,3 +350,266 @@ describe('amp-install-serviceworker', () => {
     });
   });
 });
+
+
+describes.fakeWin('url rewriter', {
+  win: {
+    location: 'https://example.com/thisdoc.amp.html',
+  },
+  amp: 1,
+}, env => {
+  let win;
+  let ampdoc;
+  let viewer;
+  let element;
+  let implementation;
+
+  beforeEach(() => {
+    win = env.win;
+    ampdoc = env.ampdoc;
+    viewer = win.services.viewer.obj;
+    element = win.document.createElement('amp-install-serviceworker');
+    element.setAttribute('src', 'https://example.com/sw.js');
+    element.setAttribute('data-no-service-worker-fallback-url-match',
+        '.amp.html');
```

Comment:
Modified this expression and added the docs. I also further test that expression is indeed parsed as a RegExp correctly.